client/main/misc/clipboardprompt.js

```diff
@@ -13,25 +13,81 @@ class ClipboardPrompt extends HTMLElement {
 
```

Comment:
while you're here, could you change the above condition to `window.navigator.platform === 'MacIntel'`